test/openshift/application.test.ts

```diff
@@ -68,7 +69,43 @@ suite('Openshift/Application', () => {
 
                 await Application.create(projectItem);
 
-                expect(result).is.equals('Empty application name');
+                expect(result).is.equals('Empty Application name');
```

Comment:
It makes no functional difference (this is just for the sake of grammar), but there is no need to combine is and equals :)
The point of the fluent API verbs here is to essentially create sentences for readability and "expect something is equals" is not really a "valid" sentence. Chai recommends using "expect something to equal", shorthand for which is "expect something equals".